frontend/components/MainPage/sections/AddFundsSection.tsx

```diff
@@ -35,13 +35,21 @@ const CustomizedCardSection = styled(CardSection)<{ border?: boolean }>`
 export const AddFundsSection = () => {
   const [isAddFundsVisible, setIsAddFundsVisible] = useState(false);
 
+  const addFunds = useCallback(async () => {
+    setIsAddFundsVisible(true);
+
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
I think I've seen it already in previous PRs, maybe make sense to implement a `delay` function so it's easier to read?